src/js/containers/search/SearchContainer.jsx

```diff
@@ -305,17 +305,7 @@ export class SearchContainer extends React.Component {
         }
 
         this.request = SearchHelper.generateUrlHash({
-            filters: {
-                ...filters,
-                tasCodes: {
-                    require: filters.tasCodes.require,
-                    exclude: filters.tasCodes.exclude
-                },
-                naicsCodes: {
-                    require: filters.naicsCodes.require,
-                    exclude: filters.naicsCodes.exclude
-                }
-            },
+            filters,
```

Comment:
Yeah I didn't realize that this endpoint has nothing to do with the request object we build for the search object. So now were passing the `counts` to this endpoint and then relying on those values, rather than recalculating the `counts` in componentDidMount. This makes it much easier.